scripts/release/update-svgs-count.js

```diff
@@ -0,0 +1,38 @@
+#!/usr/bin/env node
+/**
```

Comment:
This is unnecessary.

```suggestion
/**
```

Reply:
I think that this discussion is beyond this PR because we've the shebang in all scripts. It's true that currently are doing nothing. However, I'm inclined to give executable permissions to all scripts and maintain the shebangs, so developers can execute them directly in Linux using `./scripts/release/update-svgs-count.js`. See [this answer](https://stackoverflow.com/a/33510581/9167585) for more info about it. Do you want to open another PR for that @LitoMore?

Reply:
> Do you want to open another PR for that or an issue to discuss it

Sure.